event_model/__init__.py

```diff
@@ -2213,6 +2191,20 @@ def compose_stop(
     )(exit_status=exit_status, reason=reason, uid=uid, time=time, validate=validate)
 
 
+def dict_of_lists_has_equal_size_lists(dictionary: Dict[str, List]) -> bool:
```

Comment:
@danielballan do we support empty event pages? If so then we need to support an empty dict here.

Reply:
Yes